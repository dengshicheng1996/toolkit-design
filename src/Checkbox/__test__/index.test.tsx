import * as React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import { Checkbox } from '..'

describe('Checkbox', () => {
  it('snapshot', () => {
    const component = shallow(
      <div>
        <Checkbox checked data-y="111" /> checked
        <br />
        <Checkbox checked={false} /> no checked
        <br />
        <Checkbox checked disabled /> checked disabled
        <br />
        <Checkbox checked={false} disabled /> no checked disabled
        <br />
      </div>,
    )

    expect(toJSON(component)).toMatchSnapshot()
  })
})
