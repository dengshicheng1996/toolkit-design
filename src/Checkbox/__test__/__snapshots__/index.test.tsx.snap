// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox snapshot 1`] = `
<div>
  <Checkbox
    checked={true}
    data-y="111"
  />
   checked
  <br />
  <Checkbox
    checked={false}
  />
   no checked
  <br />
  <Checkbox
    checked={true}
    disabled={true}
  />
   checked disabled
  <br />
  <Checkbox
    checked={false}
    disabled={true}
  />
   no checked disabled
  <br />
</div>
`;
